src/data_model/task.rs

```diff
@@ -1,17 +1,15 @@
-use::uuid::Uuid;
-use::chrono::{DateTime, Local};
-struct TimeInteval {
-    start_time: DateTime<Local>,
-    end_time: DateTime<Local>,
-}
+use serde::{Deserialize, Serialize};
 
-struct Task {
-    device_id: Uuid,
-    effect: i16,
-    duration: i32,
-    interval: TimeInteval,
+#[derive(Serialize, Deserialize)]
+pub struct Timespan {
+    pub start: i64,
+    pub end: i64,
 }
 
-pub fn hello() {
-    println!("Hello World");
-}
\ No newline at end of file
+#[derive(Serialize, Deserialize)]
+pub struct Task {
+    pub id: i64,
+    pub timespan: Timespan,
+    pub duration: i64,
```

Comment:
```suggestion
    pub durationInMinuttes: i64,
```
Add the unit in the name or create a data type for it.